tests/integration_tests/functional/test_cpu_features.py

```diff
@@ -321,10 +321,10 @@ def test_cpu_rdmsr(
 
     # pylint: disable=C0121
     microvm_val_df = microvm_val_df[
-        microvm_val_df["MSR_ADDR"].isin(MSR_EXCEPTION_LIST) == False
+        ~microvm_val_df["MSR_ADDR"].isin(MSR_EXCEPTION_LIST)
```

Comment:
nit: commit messages says "use `!`", but you use `~`

Reply:
Doh! If I make a new PR I will fix it.